test/src/Semple.java

```diff
@@ -0,0 +1,7 @@
+public class Semple {
```

Comment:
@EduardKhachatryan95 Please change the letter "e" to "a" in the word Semple.

Reply:
done